### DISCUSSION TITLE
Nested operations in Formula Field

### DISCUSSION THREAD

**Discussion by tobehn** [Budibase/budibase#2654] [Help] [answered]
Can i do nested operations, like i wnat to multiply two values in a formula field and set it toFixed 2.
I tried `{{ toFixed multiply field1 field2 2}}`

The operation itself works at least i do not get any notification.

But the calculated data is all 0 or 0.0 when it has decimals in the product.

**Comment by mjashanks** [comment, marked as answer]
Hey @tobehn - you can... try this (parentheses added):

`{{ (toFixed (multiply field1 field2) 2) }}`

By the end of next month, we will be able to do these in javascript! Whether you know javascript or not, it will be much easier than the above for these sorts of expressions.

**Reply by tobehn** [reply]
Hey @mjashanks this creates an invalid syntax. 
I will wait until the next big monthly release and then use javascript for this instead.

Sounds great.

**Reply by mjashanks** [reply]
Sorry @tobehn, it's `{{ toFixed (multiply field1 field2) 2 }}` - Javascript coming at the end of October.

**Reply by tobehn** [reply]
Worked perfectly now. Thank you @mjashanks

I got you with the realease of JS in BB
